CMakeLists.txt

```diff
@@ -86,6 +86,9 @@ option(USE_EXR [=[
 Use OpenEXR while building the core openvdb library. If OPENVDB_BUILD_CORE is OFF, CMake attempts to query the located
 openvdb build configuration to decide on OpenEXR support. You may set this to on to force OpenEXR to be used if you
 know it to be required.]=] OFF)
+
+cmake_dependent_option(USE_HALF "Use the OpenEXR/half.h a half-precision floating-point library while building openvdb components. You may set this to off to remove the dependency on OpenEXR if you know you don't need float16 support." ON "USE_EXR" ON)
```

Comment:
I'm not sure if this is the right way to do this.

Reply:
This is where IlmBase/OpenEXR gets a bit weird. IIRC, half.h is part of IlmBase, *not* OpenEXR. So it's not actually dependent on OpenEXR at all. In other words, `USE_HALF` can be on or off irrespective of `USE_EXR` and vice-versa. The logic, for the core lib, would be:

- If  `USE_EXR` find both IlmBase and OpenEXR. However, if `USE_HALF` is off, disable VDB half support (no `OPENVDB_USE_HALF`)
- Otherwise if `USE_HALF`, find IlmBase only and set `OPENVDB_USE_HALF`
- Otherwise if not `USE_HALF`, do not find IlmBase or OpenEXR 

Probably worth noting that the *only* reason we depend on EXR at all outside of `half.h` (and vdb_render) is because of some functionality in `tools/RayTracer.h` :\